File: voting-backend/src/main/java/ch/rafaelurben/tamtour/voting/dto/VotingUserDto.java
package ch.rafaelurben.tamtour.voting.dto;

import java.time.OffsetDateTime;

public record VotingUserDto(
    Long id,
    String firstName,
    String lastName,
    String accountEmail,
    String accountName,
    String pictureLink,
    boolean initialRegistrationComplete,
    OffsetDateTime createdAt,
    OffsetDateTime lastLoginAt,
    boolean blocked) {}

File: voting-frontend/src/app/dto/voting-user.dto.ts
export interface VotingUserDto {
  id: number;
  firstName: string;
  lastName: string;
  accountEmail: string;
  accountName: string;
  pictureLink: string;
  initialRegistrationComplete: boolean;
  createdAt: string; // ISO datetime string including timezone
  lastLoginAt: string | null; // ISO datetime string including timezone
  blocked: boolean;
}
